### DISCUSSION TITLE
셀럽 저장 시 유튜브 채널 URL 저장에 대해

### DISCUSSION THREAD

**Discussion by shin-mallang** [woowacourse-teams/2023-celuveat#109] [Ideas]
유튜브 채널 URL은 `https://www.youtube.com/channel/`{`유뷰트 채널 ID`} 이므로 굳이 이를 저장할 필요가 없어!
지금 수정하기에는 살짝 변경해야 할 부분이 많아서, 우선 1차 개발 이후에 바꾸는 거 어때?

**Comment by odo27** [comment]
좋아요!

**Comment by kdkdhoho** [comment]
구랭 레벨4에 제거해서 리팩터링 경험하는 쪽으로 해보자~
